### DISCUSSION TITLE
Domain is already exists

### DISCUSSION THREAD

**Discussion by NaelAqel** [vercel/community#7326] [Help]
### Summary

I had a team around one year ago and they create a website for me in vercel , I lost my connection with them and I don't have access to the account they used. I stopped the website already for my hosting control panel.

I need now to implement a new website to my domain, but when I transfer my domain to vercel I had "Domain is already exists".

I don't know what to do and whom to contact, Please help me.

### Steps to Reproduce

When put transfer domain I have "Domain is already exisits".

**Comment by pawlean** [comment]
Hi @NaelAqel!

I believe that the only way to get that domain transferred to you is to ask your former team to give you access to your hosting control panel or for them to transfer to a new hosting provider.

[a comment by an automated account is not shown]
